serializable.ts

```diff
@@ -126,8 +126,7 @@ export function toPojo(
 
   if (!serializablePropertyMap) {
     throw new Error(
-      `${ERROR_MISSING_PROPERTIES_MAP}: ${context?.constructor
-        ?.prototype}`,
+      `${ERROR_MISSING_PROPERTIES_MAP}: ${context?.constructor?.prototype}`,
```

Comment:
Deno format wanted this formatted. 

Wasn't my idea, but if that's what it wants, that's what it gets.

Reply:
All good.